addons/udes_stock/models/stock_picking.py

```diff
@@ -307,6 +314,7 @@ def _compute_is_locked(self):
         """Derive is_locked from the picking's state."""
         self.is_locked = self.state != "draft"
 
+
```

Comment:
Tiniest of nits: There's a random extra new line here

Reply:
fixed